README.md

```diff
@@ -631,14 +631,14 @@ The producer has notified the consumer that the service is implemented and will
     
         assertTrue(response.getFibonacciNumbers().equals(fibonacci));
     }
-> Here we changed the name of the Test (which is not needed, but defines the test better), and used our new Request implementation to match the contract again. Note that the expected list also changed.
+> Here we changed the name of the Test (which is not a hard requirement, but describes the test better), and used our new Request implementation to match the contract again. Note that the expected list also changed.
 
 **Task 5.4** Run `mvn clean install` in the frontend module. It should compile without errors.
 
-##Conclusion
-The frontend should now be able to connect to the new numbers-service endpoint. This is the end of this workshop, but you can implement the client further to actually use the service in it's frontend. This is task 6.<br>
-Cloud Contracts provide a very useful mechanism to generate extra tests and stubs to verify the integration between services. In my opinion one of the greatest features is that the producer and the consumer are talking to the same contract and are both obligated to maintain the contract when making changes to the client or the producer. The consumer needs the changes to generate the correct stubs, while the producer needs the changes because otherwise the generated tests will fail. A possible pittfall is that the contract only needs to be changed when either side wants to make breaking changes or add functionality. When functionality or data is no longer used by a client this wont be breaking change to the stubs. This might clutter the contracts making a producer implement a contract with fields which are no longer used.
-Cloud Contracts is in no way a replacement of good team to team communications or the projects own unit tests.
+## Conclusion
+The frontend should now be able to connect to the new numbers-service endpoint. This is the end of this workshop, but you can implement the client further to actually use the service in its frontend. This is task 6.<br>
+Cloud Contracts provide a very useful mechanism to generate extra tests and stubs to verify the integration between services. One of the best features is that the producer and the consumer are talking to the same contract and are both obligated to maintain the contract when making changes to the client or the producer. The consumer needs the changes to generate the correct stubs, while the producer needs the changes because otherwise the generated tests will fail. A possible pittfall is that the contract only needs to be changed when either side wants to make breaking changes or add functionality. When functionality or data is no longer used by a client this wont be breaking change to the stubs. This might clutter the contracts making a producer implement a contract with fields which are no longer used.
+Cloud Contracts is in no way a replacement of good team to team communications or the project's own unit tests.
```

Comment:
I'm thinking you should end on a positive note. "_no way a replacement of [...] unit tests, but it goes a long way in ????_" (you're probably best off coming up with a suitable advantage yourself).